tools/c7n_azure/c7n_azure/storage_utils.py

```diff
@@ -86,7 +88,14 @@ def get_storage_keys(storage_account_id):
     def get_storage_from_uri(storage_uri):
         parts = urlparse(storage_uri)
         storage_name = str(parts.netloc).partition('.')[0]
-        container_name = parts.path.partition('/')[2]
+
+        path_parts = os.path.split(parts.path)
```

Comment:
I think if you were just to do `parts.path.split('/')., You wouldn't need to make all the `strip(...)` calls.